### ISSUE TITLE
UX testing setup for FEC conferences

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#2706]
**What we're after:** In preparation of the FEC conferences in August and October, we need to coordinate with the Information division regarding planned usability testing. 

Some things that need to be ironed out/discussed:
- [x] Get approval for updated consent form
- [ ] Advertising on training pages, during webinars leading up to the conferences, record articles, and/or announcements

**Comment by JonellaCulmer** [comment]
Before reaching out to Info, I want to get a better idea as to what we're able to do with our consent form. Currently, OGC has the consent form for review. We hope to hear back by 3/22.

**Comment by pkfec** [comment]
@JonellaCulmer have you heard from OGC on the consent form yet? Could you reach out to them to see if they have any updates?

**Comment by JonellaCulmer** [comment]
OGC has approved our consent form. I've since updated our sign up tool in ethnio with the updated consent form. 

I've reached out to the Information Division with my thoughts and requests for the user testing advertising and am now waiting for a reply.

**Comment by JonellaCulmer** [comment]
Moving to blocked while I wait for reply.

**Comment by JonellaCulmer** [comment]
I've talked to Info Division and they're going to implement some of our recommendations for advertising user testing, including 
- Include a link to your materials on our conference web pages
- Add a reference to user testing to our Record articles about the conferences

They also intend to:
- Share the email addresses of all registrants with you
- Have a designated testing table
- Mention our presence during welcoming remarks

This will evolve as we get closer to the conference. 

Closing this ticket now as it's as complete as we can get for now. We were able to discuss advertising options and that's what we hoped to get out of this work.
